ts/api/provider_frame_elem.ts

```diff
@@ -29,19 +31,38 @@ const DEFAULT_FRAME_CSS = `
   position: fixed;
   border: none;
   z-index: 9999;
+}
+
+.openyolo-visible.bottomSheet {
   bottom: 0;
+  width: 100%;
+  height: 320px;
+}
+
+.openyolo-visible.navPopout {
+  top: 0;
+  width: 100%;
+  height: 320px;
+}
+
+.openyolo-visible.fullScreen {
+  top: 0;
   left: 0;
   width: 100%;
+  height: 100%;
 }
 
-@media (min-width:801px) {
-  .openyolo-visible {
-    position: fixed;
-    left: auto;
+@media only screen and (min-device-width:321px) {
+  .openyolo-visible.bottomSheet {
```

Comment:
I like the idea of having responsive CSS, but this will make it even more difficult for providers to implement the different modes, as they will also have to implement responsive design within the IFrames. I'd rather make sure the proper mode is used according to the environment & device screen to fit exactly the need, and that we set strict standard on how each mode is rendered, so it's easier to understand and implement.